tests/trufflehog_test.py

```diff
@@ -40,6 +40,10 @@ def testTruffleHog_whenFileHasFinding_reportVulnerabilities(
     trufflehog_agent_file.process(msg)
 
     assert len(agent_mock) == 1
+    assert (
+        agent_mock[0].data["dna"]
+        == '{"location": {"android_store": {"package_name": "a.b.c"}, "metadata": [{"type": "FILE_PATH", "value": "/tmp/path/file.txt"}]}, "title": "Secret information stored in the application"}'
```

Comment:
Must include the title that was found since many secrets can be found in the same file.